srcalc/src/main/java/gov/va/med/srcalc/domain/model/Procedure.java

```diff
@@ -223,9 +223,21 @@ void setEligible(final boolean eligible)
         fEligible = eligible;
     }
 
+    /**
+     * Returns the same as {@link #getShortString()}.
+     */
     @Override
     public String toString()
     {
+        return getShortString();
+    }
+    
+    /**
+     * Returns a String in the format "cptCode - shortDesc - RVU".
+     */
+    @Transient
+    public String getShortString()
```

Comment:
Yes, that was the driver, but I didn't say that explicitly because I figured it's general-purpose enough to warrant its own method.